src/com/cft/contactmerge/io/SplitStringUtilities.java

```diff
@@ -10,23 +10,18 @@
 import java.util.*;
 
 public class SplitStringUtilities {
-    public static List<String> splitTsvString(String input)
-    {
+    public static List<String> splitTsvString(String input) {
         List<String> x = new ArrayList<String>();
 
-
-
         String testString = input;
 
-
         String _string = new String("");
         for(int i = 0;i< testString.length();i++){
             if(input.charAt(i)=='\t'){
                 x.add(_string);
```

Comment:
I think if you replace `_string` with `new String()`... it will allow you to remove both the instantiation variable, and not need to re-create a new instantiation line for every loop.